gnomenews/view.py

```diff
@@ -143,9 +148,20 @@ def update_starred_items(self):
 
     @log
     def update_feeds(self, _=None):
-        feeds = self.tracker.get_channels()
-        for new_feed in feeds:
-            if not self.feed_stack.get_child_by_name(new_feed['url']):
+        new_feeds = self.tracker.get_channels()
+        new_feed_urls = [new_feed['url'] for new_feed in new_feeds]
+        old_feed_urls = [child.get_child().get_tooltip_text() for child in self.listbox.get_children()]
+
+        # Remove old feeds
```

Comment:
Its about refreshing the UI entries, not the FeedChannels themselves - this func is called from ```GraphUpdated```